backend/corpora/common/entities/entity.py

```diff
@@ -34,13 +41,23 @@ def _query(cls, key: typing.Union[str, typing.Tuple[str, str]]) -> typing.List[B
         raise NotImplementedError()
 
     @classmethod
-    def _load(cls, db_result: typing.List[Base]) -> "Entity":
+    def _load(cls, db_result: typing.List["Entity"]) -> typing.Union["Entity", None]:
         """
-        Parses a database query result into an Entity instance
-        :param db_result: list of query result rows
+        Parses query result rows produced by Entity._query into an instantiated Entity object.
+        The output of this function is the return value of Entity.get.
+        SQLAlchemy query results are stored in a list in which each item contains Table objects returned by the query.
+        If no results are found, None is returned.
+
+        Example query results access:
+        row = query_results[0]
+        project_id = row.DbProject.id
+        :param query_results: list of query result rows
         :return: Entity
         """
-        raise NotImplementedError()
+        try:
```

Comment:
```suggestion

        try:
```
Another line break for readability :)